handlers/handlers.go

```diff
@@ -48,13 +52,13 @@ func forwardFlorenceTokenIfRequired(req *http.Request) *http.Request {
 	return req
 }
 
-//GeographyRender ...
-func GeographyRender(rend RenderClient) http.HandlerFunc {
+//GeographyHomepageRender ...
+func GeographyHomepageRender(rend RenderClient) http.HandlerFunc {
```

Comment:
same as with the other PR, probably no need to include `Render` at the end of the handler name